docs/reference/index.asciidoc

```diff
@@ -8,39 +8,45 @@ include::landing-page.asciidoc[]
 
 include::release-notes/highlights.asciidoc[]
```

Comment:
couldn't move this down - it wouldn't render

Reply:
yeah I hit that before too and I think I figured it out in meantime

just need to prepend `[chapter]` before the id, and start at `h1`, then can move it wherever thy please 🎉 

```diff
+[chapter]
 [[release-highlights]]
-== What's new in {minor-version}
+= What's new in {minor-version}
```